src/components/DopplerPlus/ExclusiveForm.js

```diff
@@ -18,6 +18,7 @@ import { useIntl } from 'react-intl';
 const ExclusiveForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } }) => {
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const [showMessage, setShowMessage] = useState();
```

Comment:
Here you could change `showMessage` by `showMessageBox`  and `setShowMessageBox` because is not a message, is a `textarea`.

Reply:
Maybe also it could be `messageBoxVisible`  and `setMessageBoxVisibility`

Reply:
changed!